doc/tutorials/dnn/dnn_yolo/dnn_yolo.markdown

```diff
@@ -29,15 +29,15 @@ Execute in webcam:
 
 @code{.bash}
 
-$ example_dnn_object_detection --config=[PATH-TO-DARKNET]/cfg/yolo.cfg --model=[PATH-TO-DARKNET]/yolo.weights --classes=object_detection_classes_pascal_voc.txt --width=416 --height=416 --scale=0.00392
+$ object_detection --config=[PATH-TO-DARKNET]/cfg/yolov3.cfg --model=[PATH-TO-DARKNET]/yolov3.weights --classes=[PATH-TO-DARKNET]/data/coco.names --width=416 --height=416 --scale=0.00392 --target=0 --device=0
```

Comment:
* A binary file has a name `example_dnn_object_detection`.
* Please do not pass default values of `target` and `device`. Keep as less parameters as possible.
* I think i'd be better to add YOLOv3 as a separate command line example. In example, run it using a [python script](https://github.com/opencv/opencv/blob/master/samples/dnn/object_detection.py).

Reply:
done